test/tests/transfers/general_field/shape_evaluation/regular/tests

```diff
@@ -136,11 +140,15 @@
       # and equidistant from 0 0 0 and 0.1 0 0, the subapp positions
       cli_args = "Outputs/file_base=nearest_subapp Transfers/active='from_sub from_sub_elem'
           Transfers/from_sub/use_nearest_app=true Transfers/from_sub_elem/use_nearest_app=true
+          Transfers/from_sub/bbox_factor=1 Transfers/from_sub_elem/bbox_factor=1
           MultiApps/sub/positions='0 0 0 0.1 0 0'
           Transfers/from_sub/search_value_conflicts=true Transfers/from_sub_elem/search_value_conflicts=true
           MultiApps/sub/cli_args='Mesh/xmin=-0.4;Mesh/xmax=0.8;Mesh/ymax=0.7'"
-      expect_err = "multiple valid values from equidistant points were "
+      expect_err = "(Search for nearest position found several matches|multiple valid values from "
+                   "equidistant points were )"
       detail = "when using the closest app rather than the first valid value to chose a value"
+      method = !opt
```

Comment:
why this restriction?

Reply:
check on who's closest is only done in devel for nearest position.

I ve been thinking and I ll change that. NN is just too dangerous

Reply:
has been changed